### PR TITLE
Fix unstable test

### PR DESCRIPTION
**Pull request by erikjv**

### CODE REVIEW

File: test/testallfilesdeleted.cpp [hunk 1]
```diff
@@ -300,8 +300,10 @@ private slots:
     }
 
     void testSelectiveSyncNoPopup() {
+        auto original = FileInfo::A12_B12_C12_S12();
```
Comment (TheOneRing) [inline test/testallfilesdeleted.cpp:303]: I understand the original is not to be modified, so maybe const?

**Review by TheOneRing — COMMENTED** [review]
Good catch while including mtime in the tests proofs to be a bit of a pain I'm pretty sure it improved the quality of the tests.

File: test/testutils/syncenginetestutils.h [hunk 2]
```diff
@@ -628,7 +628,7 @@ inline void addFiles(QStringList &dest, const FileInfo &fi)
         for (const auto &fi : fi.children)
             addFiles(dest, fi);
     } else {
-        dest += QStringLiteral("%1 - %2 %3-bytes").arg(fi.path()).arg(fi.contentSize).arg(fi.contentChar);
+        dest += QStringLiteral("%1 - %2 %3-bytes (%4)").arg(fi.path()).arg(fi.contentSize).arg(fi.contentChar).arg(fi.lastModifiedInUtc());
```
Comment (TheOneRing) [inline test/testutils/syncenginetestutils.h:631]: .arg(, , , , )

Also the build seems to fail with this one.

File: test/testallfilesdeleted.cpp (hunk 1 above)
Comment (erikjv) [reply test/testallfilesdeleted.cpp]: done

File: test/testutils/syncenginetestutils.h (hunk 2 above)
Comment (erikjv) [reply test/testutils/syncenginetestutils.h]: done

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**